plugins/scaffolder/src/alpha/components/TemplateEditorPage/TemplateEditorLayout.tsx

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { PropsWithChildren } from 'react';
+import { WithStyles, withStyles } from '@material-ui/core/styles';
+
+export const TemplateEditorLayout = withStyles(
+  theme => ({
+    root: {
+      height: '100%',
+      gridArea: 'pageContent',
+      display: 'grid',
+      gridTemplateAreas: `
+      "toolbar"
+      "browser"
+      "editor"
+      "preview"
+      "results"
+    `,
+      [theme.breakpoints.up('md')]: {
+        gridTemplateAreas: `
+      "toolbar toolbar toolbar"
+      "browser editor preview"
+      "results results results"
+    `,
+        gridTemplateColumns: '1fr 3fr 2fr',
+        gridTemplateRows: 'auto 1fr auto',
+      },
+    },
+  }),
+  { name: 'ScaffolderTemplateEditorLayout' },
+)(({ children, classes }: PropsWithChildren<WithStyles>) => (
+  <main className={classes.root}>{children}</main>
+));
+
+export const TemplateEditorLayoutToolbar = withStyles(
+  {
+    root: {
+      gridArea: 'toolbar',
+    },
+  },
+  { name: 'ScaffolderTemplateEditorLayoutToolbar' },
```

Comment:
Hmm, would kinda prefer these to be under `ScaffolderTemplateEditorLayout` in the `toolbar` slot etc, like they were before.

Reply:
Do you mean like this?https://github.com/backstage/backstage/pull/26917/commits/6107337fa69a9c572c72e039f766a41bade079cf

Reply:
As discussed, let's leave as is since this is by far the closest to the desired state for MUI 5. We'll want to break these names when migrating, but that's a lil bit inevitable anyway